packages/components/Separator/src/Separator.types.ts

```diff
@@ -1,11 +1,27 @@
 import { ViewProps, ColorValue } from 'react-native';
 
-interface SeparatorPropTokens {
+export const separatorName = 'Separator';
+
+export const SeparatorInsetSpacings = [0, 16, 56, 68, 72, 108] as const;
+
+export type SeparatorInsetSpacing = typeof SeparatorInsetSpacings[number];
+
+export interface SeparatorPropTokens {
   /**
    * Specifies whether the separator is vertical or horizontal. If true, then it's vertical.
    * @defaultValue 'false'
    */
   vertical?: boolean;
+
+  /**
+   * Inset Spacing in pixels.
```

Comment:
I don't think. this is in pixels in iOS. Instead it would be on "points", (1 pixel at 1x DPI, 3 pixels in 3X DPI...). I think that might be true on Android as well. Worth confirming?

Reply:
On Android, it's probably DP https://developer.android.com/training/multiscreen/screendensities

Reply:
Changed. 
Android - dp 
iOS - points